lib/ukcpio/cpio.c

```diff
@@ -70,7 +101,13 @@ extract_file(const char *path, const char *contents, size_t len,
 
 	uk_pr_info("Extracting %s (%zu bytes)\n", path, len);
 
-	fd = uk_syscall_r_open(path, O_CREAT | O_WRONLY | O_TRUNC, 0);
+	fd = uk_syscall_r_open(path, O_CREAT | O_WRONLY | O_EXCL, 0);
+	if (fd == -EEXIST) {
+		if ((err = try_remove(path)))
+			uk_pr_warn("Path cleanup failed: %s (%d)\n",
+				   strerror(-err), -err);
+		fd = uk_syscall_r_open(path, O_CREAT | O_WRONLY | O_TRUNC, 0);
```

Comment:
Let's also add an additional message here, so that we know when a try remove + open happened.